src/main/java/com/mcmoddev/mmdbot/core/BotConfig.java

```diff
@@ -365,20 +400,31 @@ public long getCommunityChannelCategory() {
             return permissions;
         }
         MMDBot.LOGGER.warn("Unknown format of \"community_channels.owner_permissions\", resetting to blank list");
```

Comment:
This should also be modified to use the new `COMMUNITY_CHANNEL_OWNER_PERMISSIONS` constant.

Reply:
Should be fixed.